src/model/Post.js

```diff
@@ -0,0 +1,102 @@
+import {EventEmitter} from "events";
```

Comment:
Because you are exporting an object (and not a class), you should have named this file `post.js` instead of `Post.js`. You don't have to change it now (especially because Windows don't not treat case changes in file names correctly...